src/objects/core/admin.py

```diff
@@ -62,6 +63,8 @@ class ObjectRecordInline(GeoModelAdminMixin, admin.TabularInline):
         "correct",
     )
 
+    formfield_overrides = {GeometryField: {"widget": forms.Textarea}}
```

Comment:
Do we know what format you have to use to enter coordinates? Might be good to have an example in the help text if we do